Mathlib/NumberTheory/DiophantineApproximation.lean

```diff
@@ -597,12 +594,11 @@ theorem exists_rat_eq_convergent {q : ℚ} (h : |ξ - q| < 1 / (2 * (q.den : ℝ
 /-- The main result, *Legendre's Theorem* on rational approximation:
 if `ξ` is a real number and `q` is a rational number such that `|ξ - q| < 1/(2*q.den^2)`,
 then `q` is a convergent of the continued fraction expansion of `ξ`.
-This is the version using `generalized_contined_fraction.convergents`. -/
-theorem exists_continued_fraction_convergent_eq_rat {q : ℚ}
-    (h : |ξ - q| < 1 / (2 * (q.den : ℝ) ^ 2)) :
-    ∃ n, (GeneralizedContinuedFraction.of ξ).convergents n = q := by
+This is the version using `GenContFract.convs`. -/
+theorem exists_genContFract_convs_eq_rat {q : ℚ}
```

Comment:
Ah, sorry. Maybe I should have been more explicit in my final comment. This theorem also should have been renamed to be consistent with `convs_eq_convergent`, so `exists_convs_eq_rat`.

Reply:
(Resolved in #13555)